### PR TITLE
uyuni server migration legacy to container

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/24769 **with link to rendered HTML output.**

### CODE REVIEW

**Review by raulillo82 — APPROVED** [review]
LGTM. But maybe better to wait for a second review.

**Review by aaannz — APPROVED** [review]

**Comment by keichwa** [comment]
> LGTM. But maybe better to wait for a second review.

Ondrej also approved it, but according to the log here, I removed the check mark ;-(

**Review by aaannz — APPROVED** [review]

**Pull request merged**
